README.md

```diff
@@ -439,6 +473,25 @@ If your certificate or private key have different names, you can set the environ
 
 For letsencrypt certs, we'll autodetect that and add the needed Identrust X3 CA Cert automatically. In case your letsencrypt cert is already the chained certificate, you can set the `CERT_IS_CHAIN` environment variable to `true`, e.g. `CERT_IS_CHAIN=true`. This option also works together with a custom `CERTNAME`.
 
+~~### multiarch~~
```

Comment:
This section is relevant with the current tags, and specifically the stuff about `armhf` going away needs notice. I suspect a  new release of unifi will mandate `armhf` support being  removed within the next year.

Reply:
I will move it back below the Tags section